src/role.builder.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import . "screepsgo/src/screeps"
+
+func roleBuilder(creep Creep) {

```

Comment:
I think that the function names for roleX should be changed to reflect their behaviour better. What exactly do they do? Can we split them into smaller tasks and does it make sense?

Reply:
Bot logic. See "harvester" thread.